build_mco.sh

```diff
@@ -3,15 +3,21 @@ help() {
     echo ""
     echo "Usage: ./build_mco.sh [options] <quay.io username>"
     echo "Options:"
-    echo "-h        show this message"
+    echo "-h, --help      show this message"
+    echo "-t, --tag       push to a custom tag in your origin release image repo, default: mco"
     echo ""
 }
 
+TAG="mco"
+
 # Parse Options
 case $1 in
     -h|--help)
         help
         exit 0;;
+    -t|--tag)
+        TAG=$2
+        shift;;
```

Comment:
you need to shift twice otherwise USERNAME below will resolve to the tag.
But really, if we keep adding more options to this script we should consider using getops

Reply:
ah oops, haha fair enough.